docs/how-to-use-gvfs.md

```diff
@@ -42,7 +42,10 @@ the path mapping and convert automatically.
 
 ### Prerequisites
 
-+ A Hadoop environment with HDFS or other Hadoop Compatible File System (HCFS) implementations like S3, GCS, etc. GVFS has been tested against Hadoop 3.3.1. It is recommended to use Hadoop 3.3.1 or later, but it should work with Hadoop 2.x. Please create an [issue](https://www.github.com/apache/gravitino/issues) if you find any compatibility issues.
++ A Hadoop environment with HDFS running. GVFS has been tested against
```

Comment:
> A Hadoop environment with HDFS running.

Is this a GVFS prerequisite?

Reply:
<img width="1148" alt="image" src="https://github.com/user-attachments/assets/7f4aa0a4-0a65-4765-8e58-f0a33339652c" />
This line was added by @justinmclean , Indeed, the description is not correct.

Reply:
I have removed this sentence.